src/base/net/smtp.cpp

```diff
@@ -525,3 +526,12 @@ QString Smtp::getCurrentDateTime() const
     QString ret = weekDayStr + ", " + dayStr + " " + monthStr + " " + yearStr + " " + timeStr + " " + timeOffsetStr;
     return ret;
 }
+
+void Smtp::error(QAbstractSocket::SocketError socketError)
+{
+    // Getting a remote host closed error is apparently normal, even when successfully sending
+    // an email
+    if (socketError != QAbstractSocket::RemoteHostClosedError) {
```

Comment:
As per our [coding guidelines](https://github.com/qbittorrent/qBittorrent/blob/master/CODING_GUIDELINES.md) remove the curly braces here.